### PR TITLE
WFLY-8536 WFLY-8537 Unmanaged transitive dependencies in EJB client B…

### PR DESCRIPTION
**Pull request by ctomc**
…OM dependency tree

### CODE REVIEW

**Comment by ctomc** [comment]
@psakar please review

**Comment by kabir** [comment]
Retest this please

**Comment by kabir** [comment]
Retest this please

File: feature-pack/src/main/resources/content/docs/licenses/licenses.xml [hunk 1]
```diff
@@ -3853,6 +3853,17 @@
       </licenses>
     </dependency>
     <dependency>
+      <groupId>>org.wildfly.client</groupId>
```
Comment (dmlloyd) [inline feature-pack/src/main/resources/content/docs/licenses/licenses.xml]: Typo here

**Pull request merged**